x/wasm/internal/keeper/test_common.go

```diff
@@ -319,9 +335,9 @@ func handleExecute(ctx sdk.Context, k *Keeper, msg *types.MsgExecuteContract) (*
 	return res, nil
 }
 
-func AnyAccAddress(_ *testing.T) sdk.AccAddress {
+func AnyAccAddress(_ *testing.T) string {
```

Comment:
If `AccAddress` is a binary address, isn't the funtion name misleading now? What about `func RandomAddress`?

Reply:
Ja, it is a random account address in bech32 format. So `RandomBech32AccountAddress`